jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -108,6 +108,11 @@ else
   abort("Unknown 'forwarded_client_cert' option: #{forwarded_client_cert_option}. Known options: 'always_forward_only', 'forward_only', 'sanitize_set', 'forward_only_if_route_service'")
 end
 
+disable_domain_fronting = p("ha_proxy.disable_domain_fronting")
+if ![true, false, "mtls_only"].include?(disable_domain_fronting)
```

Comment:
hmm does this really work? can `disable_domain_fronting` be a boolean and a string at the same time?

Reply:
Fixed